models/endpoints.model.js

```diff
@@ -0,0 +1,10 @@
+const fs = require("fs/promises");
+
+module.exports.fetchAvailableEndpoints = () => {
+    return fs
+      .readFile("./endpoints.json", "utf-8")
+        .then((data) => {
+           const endpoints =  JSON.parse(data);
+            return endpoints;
+        })
+};
```

Comment:
Good work, using fs readfile here is good.

Tip: `exports.fetchAvailableEndpoints` is fine as a shorthand here instead of module.exports